doc/man3/OPENSSL_LH_stats.pod

```diff
@@ -61,7 +61,7 @@ These calls should be made under a read lock. Refer to
 L<OPENSSL_LH_COMPFUNC(3)/NOTE> for more details about the locks required
 when using the LHASH data structure.
 
-The functions OPENSSH_LH_stats() and OPENSSH_LH_stats_bio() were deprecated in
+The functions OPENSSL_LH_stats() and OPENSSL_LH_stats_bio() were deprecated in
 version 3.2.
```

Comment:
Seeing this ... they are actually deprecated since 3.1 and we need to update the deprecation notice in the master branch ASAP - before the beta.

Reply:
I am going to do a fix in master branch for that and I'll include the typo fix. Could you please instead change your typo fix PR to be against 3.1?